Add minor ticks on and off in Axis (#22761)
matplotlib/matplotlib: doc/users/next_whats_new/axis_minorticks_toggle.rst

Toggle minorticks on Axis
------------------------------

> Similar to Axes, minor ticks on Axis can be displayed or removed
> using ``minorticks_on()`` and ``minorticks_off()``.

Reviewer: ```suggestion
Minor ticks on an `~.Axis` can be displayed or removed using `~.Axis.minorticks_on` 
and `~.Axis.minorticks_off`; eg: ``ax.xaxis.minorticks_on()``.  See also `~.Axes.minorticks_on`.  
```

> using ``minorticks_on()`` and ``minor_ticks_off()``.

Reviewer 2: ```suggestion
using ``minorticks_on()`` and ``minorticks_off()``.
```

Author: Fixed.
